package.json

```diff
@@ -22,7 +22,8 @@
         "sprites": "create-svg-sprite --optimize --input './svg' --output './svg' --name 'icons' && create-svg-sprite --optimize --input './symbols' --output './symbols' --name 'symbols'",
         "build": "yarn && yarn copy && rm -r iconfont && mkdir iconfont && icon-font-generator svg/*.svg -o iconfont -n PXBlueIcons -p pxb && yarn sprites && yarn index",
         "build:progress": "cd progress/react && yarn && yarn build && cd ../angular && yarn && yarn build",
-        "build:all": "yarn && yarn build && yarn build:progress"
+        "build:all": "yarn && yarn build && yarn build:progress",
+        "start:ng-progress-icons": "./scripts/linkProgressAngular.sh && cd progress/angular && yarn start"
```

Comment:
I would move this command into the /progress folder and call it `yarn:start-angular` to match convention. This top-level is like mission control for things that affect all of the packages below, while this is specific to the progress icons package.

Reply:
into the progress folder? progress folder currently doesn't have any package.json. Should I create a new one?